europe/README.EUROPE

```diff
@@ -457,4 +457,26 @@ only used as steps in more complicated transformations.
 
 Init file with various transformations and conversions of relevance in Iceland.
 
-* ISL
\ No newline at end of file
+* ISL
+
+### Hungary: ETRS89 HD72/EOV (EPSG:23700)
+
+*Source*: [Department of Geodesy and Surveying of Budapest University of Technology and Economics](https://github.com/OSGeoLabBp/eov2etrs)  
+*Format*: NTv2
+*License*: [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/)
+
+Grid transformation from ETRS89 to HD72/EOV.
```

Comment:
- is it offsets from ETRS89 to HD72 ? This is slightly unusual, and against the EPSG practice to go from "local" to "global" CRS
- hum, HD72/EOV is a projected CRS.... Don't you mean the geographic 2D CRS EPSG:4237 instead ?

Reply:
I've tried to compare with the Helmert-based transformation included in EPSG, EPSG:1449, HD72 to ETRS89 (2)
```
$ echo "20 47 0" | src/cct -d 8 +proj=pipeline +proj=unitconvert +xy_in=deg +xy_out=rad +step +proj=push +v_3 +step +proj=cart +ellps=GRS67 +step +proj=helmert +x=52.684 +y=-71.194 +z=-13.975 +rx=0.312 +ry=0.1063 +rz=0.3729 +s=1.0191 +convention=coordinate_frame +step +inv +proj=cart +ellps=GRS80 +step +proj=pop +v_3 +step +proj=unitconvert +xy_in=rad +xy_out=deg
  19.99887740    46.99973951    0.00000000           inf
```
I do get a compatible result when using
```
$ echo "20 47 0"  | src/cct -d 8  +proj=pipeline +step +proj=hgridshift +grids=etrs2eov_notowgs.gsb
   19.99887893     46.99973967    0.00000000           inf
```
So this means that this grid is from HD72 to ETRS89. It should likely be renamed so that it is obvious

Reply:
The offsets are between EPSG:23700 and EPSG_4258. The HD72/EOV refers to both the datum and projection system. I don't why this name is used. The cs2cs command is:
 `cs2cs -f "%.7f" +init=epsg:23700 +nadgrids=etrs2eov_notowgs.gsb +geoidgrids=geoid_eht2014.gtx +to +init=epsg:4258`

Reply:
> he offsets are between EPSG:23700 and EPSG_4258

That doesn't make sense from a modelization point of view. A NTv2 grid is the correction between 2 geographic CRS. Of course it can be (indirectly) applied to a projected CRS based on them, such as EPSG:23700 is based upon EPSG:4237, but from a documentation point of view, only the geographic CRS must be documented. That is this grid is from EPSG:4237 (HD72) to EPSG:4258 (ETRS89)